tests/xtro-sharpie/macOS-WebKit.ignore

```diff
@@ -593,3 +593,28 @@
 !missing-type! WebUndefined not bound
 !unknown-native-enum! WKSelectionGranularity bound
 !wrong-enum-size! WebNavigationType managed 4 vs native 8
+
+# TODO Below are marked as macos(WK_MAC_TBA) as of 11.4 beta2
+!missing-protocol-conformance! WKWebView should conform to NSTextFinderClient (defined in 'WKNSTextFinderClient' category)
+!missing-selector! WKFindConfiguration::backwards not bound
+!missing-selector! WKFindConfiguration::caseSensitive not bound
+!missing-selector! WKFindConfiguration::setBackwards: not bound
+!missing-selector! WKFindConfiguration::setCaseSensitive: not bound
+!missing-selector! WKFindConfiguration::setWraps: not bound
+!missing-selector! WKFindConfiguration::wraps not bound
+!missing-selector! WKFindResult::matchFound not bound
+!missing-selector! WKPDFConfiguration::rect not bound
+!missing-selector! WKPDFConfiguration::setRect: not bound
+!missing-selector! WKWebView::createPDFWithConfiguration:completionHandler: not bound
+!missing-selector! WKWebView::createWebArchiveDataWithCompletionHandler: not bound
+!missing-selector! WKWebView::findString:withConfiguration:completionHandler: not bound
+!missing-selector! WKWebView::pageZoom not bound
+!missing-selector! WKWebView::printOperationWithPrintInfo: not bound
+!missing-selector! WKWebView::setPageZoom: not bound
+!missing-type! WKFindConfiguration not bound
+!missing-type! WKFindResult not bound
+!missing-type! WKPDFConfiguration not bound
+
+# TODO mediaType is not available in 11.4 beta2 even though it is marked as such
```

Comment:
Not sure whether we normally create GH issues for this/the TODO above? I will create issues if that's the standard practice :)

Reply:
In general we don't need issue if it's in the `.ignore` file - since it's already a list of issues

However mismatch between Apple's header and implementation should be filled with Apple (and you create a shadow issue in maccore for tracking). That way whenever it's fixed we'll know what needs to be done on our side to close the loop.

Reply:
@whitneyschmidt example of such an issue: https://github.com/xamarin/maccore/issues/2065